### PR TITLE
feature/PLU-4180-sort-api-resource-operations

### PR DESCRIPTION
**Pull request by alexandrtspl**
* sort ApiResourceOperations

### CODE REVIEW

File: config/rector/eonx-set.php [hunk 1]
```diff
@@ -50,6 +53,14 @@
     $services->set(RemoveNonExistingVarAnnotationRector::class);
     $services->set(RestoreDefaultNullToNullableTypeParameterRector::class);
     $services->set(RestoreDefaultNullToNullableTypePropertyRector::class);
+    $services->set(SortApiResourceOperationsRector::class)->call('configure', [
```
Comment (ERuban) [inline config/rector/eonx-set.php:56]: Not sure that we need it in main set.
Not all projects that uses this set works with api-platform - as example Points_v1 (https://github.com/eonx-com/points-api-v1/blob/release/src/rector.php)

File: tests/Rector/SortApiResourceOperationsRector/Fixture/ApiResourceOperationsSorted.php.inc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        'another_custom_opration' => 'another_custom_operation',
+        'some_custom_operation' => [
+            'some_custom_attr' => 'some_custom_value',
+        ],
+    ],
+    normalizationContext: ['groups' => ['...']]
+)]
+class SomeClass
+{
+}
```
Comment (ERuban) [inline tests/Rector/SortApiResourceOperationsRector/Fixture/ApiResourceOperationsSorted.php.inc:47]: Please, add empty line at the and of the class.

File: src/Rector/EasyRankeable/ApiResourceOperationEasyRankeable.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    /**
+     * An array to sort the element order by
+     *
+     * @return bool[]|int[]
+     */
+    public function getRanks(): array
+    {
+        return [
+            $this->getName() !== 'get',
```
Comment (ERuban) [inline src/Rector/EasyRankeable/ApiResourceOperationEasyRankeable.php:32]: Why those standard operations sorted like this (not-alphabetically)?
And also do we really need those `activate` and `deactivate` operations to be on the top of other non-standard operations?

**Review by ERuban — CHANGES_REQUESTED** [review]

File: config/rector/eonx-set.php (hunk 1 above)
Comment (alexandrtspl) [reply config/rector/eonx-set.php]: I'm agreed. Removed

File: tests/Rector/SortApiResourceOperationsRector/Fixture/ApiResourceOperationsSorted.php.inc (hunk 2 above)
Comment (alexandrtspl) [reply tests/Rector/SortApiResourceOperationsRector/Fixture/ApiResourceOperationsSorted.php.inc]: Added

**Review by ERuban — APPROVED** [review]

**Review by akl-95 — APPROVED** [review]

**Pull request closed without merging**